setup.cfg

```diff
@@ -64,3 +70,5 @@ where = src
 [options.entry_points]
 napari.manifest =
     napari-n2v = napari_n2v:napari.yaml
+napari.plugin =
+    napari-n2v2 = napari_n2v.utils.n2v_utils
```

Comment:
I would not call it n2v2 since this corresponds to a version of N2V with modified architecture ([paper here](https://openreview.net/forum?id=IZfQYb4lHVq)).

Where is the name used?

Reply:
Nowhere, you can also call it n2v_

Reply:
@haesleinhuepf I was going over the code again for DenoiSeg, and something I didn't notice before is that the `entry_points` points to `napari_n2v.utils.n2v_utils`. That doesn't make sense to me. Where is that used?

Shouldn't it be where `@register_function(menu="Filtering / noise removal > Apply N2V denoiser")` is called? (wild guess since I don't know where the `entry_points` is defined.)

Reply:
This is npe1, yes. It doesn't matter so much where it points at as long as the `@register...` code blocks are imported from there...